pkgs/development/python-modules/ajsonrpc/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, isPy3k, buildPythonPackage, fetchPypi, pytestCheckHook }:
+
+buildPythonPackage rec {
+  pname = "ajsonrpc";
+  version = "1.1.0";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1b5r8975wdnk3qnc1qjnn4lkxmqcir3brbwnxml9ii90dnsw408a";
+  };
+
+  checkInputs = [ pytestCheckHook ];
```

Just an additional safeguard.

```suggestion
  checkInputs = [ pytestCheckHook ];
  
  pythonImportsCheck = [ "ajsonrpc" ];
```